ouroboros-consensus/src/unstable-consensus-testlib/Test/Ouroboros/Consensus/QuickCheck/Extras.hs

```diff
@@ -1,4 +1,4 @@
-module Test.QuickCheck.Extras (
+module Test.Ouroboros.Consensus.QuickCheck.Extras (
```

Comment:
This module needs to be moved, as `quickcheck-dynamic` also has `Test.QuickCheck.Extras`.